SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DAQmx/Types.cs

```diff
@@ -53,6 +53,13 @@ public class AOFunctionGenerationSettings
         /// This value is used to get/set the value of the NationalInstruments.DAQmx.AOChannel.FunctionGenerationDutyCycle.
         /// </remarks>
         public double? DutyCycle { get; set; }
+
+        /// <summary>
+        /// Set up the configurations for an Analog Output Function Generation task in DAQmx.
```

Comment:
So the configurations are not set just using the default constructor.

How about "Creates an object where configurations for an Analog Output Function Generation DAQmx task could be set."?

@Mattjet27 any suggestions on the wording?

Reply:
@lornazh, my suggestion would be: "Creates an object to define and set Analog Output Function Generation specific settings. This object is used as an parameter for configuring an Analog Output Function Generation DAQmx task <see cref="AnalogOutputFunctionGeneration.ConfigureAOFunctionGeneration(DAQmxTasksBundle, AOFunctionGenerationSettings)"/>".

Reply:
Updated